run.sh

```diff
@@ -2,6 +2,14 @@
 
 { # this ensures the entire script is downloaded #
 
+printf "\n\nThis script requires an npm configuration that allows global\n"
+printf "installs without sudo. We'll be checking whether you have globally\n"
+printf "installed packages and if so, we'll be moving them to a new folder.\n\n"
+printf "You may have to type in your password once to allow moving all\n"
+printf "your currently installed packages.\n\n"
+
+./scripts/npm-no-sudo.sh
```

Comment:
I'd add a `curl` line similar to
```
bash <(curl -s https://raw.githubusercontent.com/Wolox/react-bootstrap/development/run.sh) [--verbose]
```
instead, since while executing the first `curl`, `./scripts/npm-no-sudo.sh` may not exist locally.

Reply:
Sure thing! Although, wouldn't it be better to have the script bundled within this repo?